xbmc/interfaces/legacy/ListItem.h

```diff
@@ -222,8 +222,9 @@ namespace XBMCAddon
        *     - lastplayed    : string (%Y-%m-%d %h:%m:%s = 2009-04-05 23:16:04)
        *     - album         : string (The Joshua Tree)
        *     - artist        : list (['U2'])
-       *     - votes         : string (12345 votes)
+       *     - votes         : integer (12345)
        *     - trailer       : string (/home/user/trailer.avi)
+       *     - date          : string (%Y-%m-%d %h:%m:%s = 2009-04-05 23:16:04)
```

Comment:
for episodes, both return the same value indeed.

there are various use cases of the 'date' infolabel, depending on the context:
tv shows: first aired date of a tvshow
pictures: date/time the picture was taken
file: date/time stamp of the file